src/model/OWL.pl

```diff
@@ -382,13 +406,13 @@
   is_all_disjoint_classes(DC),
   tripledb_ask(DC,owl:members,RDF_list),
   is_rdf_list(RDF_list,List),
-  once((
+  (
     member(Sup_A,List), subclass_of(A,Sup_A),
     member(Sup_B,List), unify_disjoint_(B,Sup_B)
-  )).
+  ).
 
 unify_disjoint_(B,Disjoint) :-
-  ( var(B) -> B=Disjoint ; subclass_of(B,Disjoint) ).
+  ( B=Disjoint ; transitive(subclass_of(B,Disjoint)) ).
```

Comment:
If B is unbound and Disjoint is bound then `transitive(subclass_of(B,Disjoint))` will not return all Subclasses of Disjoint. What is the best way to get the transitive closure of all subclasses of a given class?

Reply:
there is a predicate `transitive/1`, you can call it with some goal as argument such as `subclass_of(A,B)`.
for subclass-of, this is just doing one DB query to find the transitive closure

Reply:
hmm, if this is the case, it might be a bug in the triple DB client for that case.

Reply:
Added a fixme to the corresponding tests